### PR TITLE
chore: add prerelease script to copy README

### PR DESCRIPTION
**Pull request by MarsiBarsi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "start": "nx run demo:serve",
     "build": "nx affected --target build --prod --with-deps",
     "release": "nx affected --target release",
+    "release:package": "nx build ${npm_config_package} --prod && npm run sync-readmes && nx semrel ${npm_config_package}",
```
Comment (IKatsuba) [inline package.json]: Давай синкать readme тоже точечно. Какой пакет передали, тот и синкакаем. Избавимся от хардкодного списка библиотек в скрипте

Comment (MarsiBarsi) [reply package.json]: о, ты прав!

**Review by IKatsuba — APPROVED** [review]

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version core@0.1.0-alpha.5 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version webpack-plugin@0.1.0-alpha.6 :tada:

The release is available on [GitHub release](<github_release_url>)

**Comment by IKatsuba** [comment]
:tada: This pull request is included in version angular@0.1.0-alpha.8 :tada:

The release is available on [GitHub release](<github_release_url>)

**Pull request merged**